WORKSPACE

```diff
@@ -1,32 +1,31 @@
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
-# To update TensorFlow to a new revision,
+# To update OpenXLA to a new revision,
 # a) update URL and strip_prefix to the new git commit hash
 # b) get the sha256 hash of the commit by running:
-#    curl -L https://github.com/tensorflow/tensorflow/archive/<git hash>.tar.gz | sha256sum
+#    curl -L https://github.com/openxla/xla/archive/<git hash>.tar.gz | sha256sum
 #    and update the sha256 with the result.
 http_archive(
-    name = "org_tensorflow",
-    patch_args = [
-        "-l",
-        "-p1",
-    ],
-    patch_tool = "patch",
-    patches = [
-        "//tf_patches:cache_urls.diff",
-        "//tf_patches:f16_abi_clang.diff",
-        "//tf_patches:gpu_race_condition.diff",
-        "//tf_patches:grpc_version.diff",
-        "//tf_patches:stream_executor.diff",
-        "//tf_patches:thread_local_random.diff",
-        "//tf_patches:xplane.diff",
-        "//tf_patches:topk_rewriter.diff",
-        "//tf_patches:local_rendezvous.diff",
-        "//tf_patches:triton_filesystem.diff",
-    ],
-    strip_prefix = "tensorflow-5e229cfaa3c13c389f864b0ed38877fb61936161",
+    name = "xla",
+    # patch_args = [
+    #     "-l",
+    #     "-p1",
+    # ],
+    # patch_tool = "patch",
+    # patches = [
+    #     "//openxla_patches:cache_urls.diff",
```

Comment:
why is this section commented out?

Reply:
I didn't patch PTXLA with patches, because I didn't check patches, but we build successfully, so I skipped this patch part and commented out it